### PR TITLE
[OKTA-461263] Google authenticator integration guide (.NET SDK)

### PR DESCRIPTION
**Pull request by danielmaharry-okta**
## Description:
- **What's changed?** Nitfixes for the Google Authenticator integration guide for dotnet but also a new PR for people to review the guide if they haven't done so already.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-461263](https://oktainc.atlassian.net/browse/OKTA-461263)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
 * Build a sign-in page on the client
 * Authenticate the user credentials
 * Handle the response from the sign-in flow
 * Display a list of possible authenticator factors
 
 ### 5: Check Authenticator Status
-When the user selects the Google Authenticator factor and clicks **Submit**, the form posts back to the `SelectAuthenticatorAsync` method. This checks whether the user is in Challenge Flow or Enrollment Flow.
+
+When the user selects Google Authenticator to authenticate themselves and clicks **Submit**, the form posts back to the `SelectAuthenticatorAsync` method. This checks whether the user is in Challenge Flow or Enrollment Flow.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md:12]: "This checks whether the user is in Challenge Flow or Enrollment Flow." - consider rewriting the sentence as it's somewhat fragmented.

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md [hunk 2]
```diff
@@ -21,7 +23,7 @@ selectAuthenticatorResponse = await _idxClient.SelectChallengeAuthenticatorAsync
     selectAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification` and the Challenge Page should be displayed.
+If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification` and the Challenge page should be displayed.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md:26]: comma after `AwaitingAuthenticatorVerification`?

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
Looks great Dan! I provided some feedback items.

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md (hunk 1 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md]: Have rewritten this a bit.

**Review by robertnicolo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md [hunk 3]
```diff
@@ -21,7 +23,7 @@ selectAuthenticatorResponse = await _idxClient.SelectChallengeAuthenticatorAsync
     selectAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification` and the Challenge Page should be displayed.
+If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification`, and the Challenge page should be displayed.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md:26]: Display is transitive and requires an object to complete the meaning. Appears is intransitive and doesn't require an object to complete the meaning.

```suggestion
If the call is successful, the returned `selectAuthenticatorResponse` object has an `AuthenticationStatus` of `AwaitingAuthenticatorVerification`, and the Challenge page should appear.
```

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/index.md [hunk 4]
```diff
@@ -39,13 +39,13 @@ Google Authenticator is an authenticator app developed by Google used to verify
 
 ### Authentication flow
 
-Once a user is enrolled in Google Authenticator, they are challenged by the service provider (for example website) to provide a Time-based One-Time Password (TOTP) during authentication. Google Authenticator generates the one-time password, which is submitted by the user to the service provider for verification. The service provider independently generates the password and validates that the submitted password is identical to the generated one.
+Once a user is enrolled in Google Authenticator, they are challenged by the service provider (for example, a website) to provide a Time-based One-Time Password (TOTP) during authentication. Google Authenticator generates the TOTP, which is submitted by the user to the service provider for verification. The service provider independently generates the password and validates that the submitted password is identical to the generated one.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/index.md:42]:
```suggestion
After a user is enrolled in Google Authenticator, they are challenged by the service provider (for example, a website) to provide a Time-based One-Time Password (TOTP) during authentication. Google Authenticator generates the TOTP, which is submitted by the user to the service provider for verification. The service provider independently generates the password and validates that the submitted password is identical to the generated one.
```

**Review by susanharper-okta — APPROVED** [review]
couple nits

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md (hunk 3 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/challengeintegrationsteps.md:26]: fixed

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/index.md (hunk 4 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/index.md:42]: Fixed

**Review by susanharper-okta — APPROVED** [review]
1 typo - approving to not hold you up.

**Pull request merged**